code/modules/cargo/packs.dm

```diff
@@ -465,7 +472,9 @@
 					/obj/item/tank/internals/plasma,
 					/obj/item/grenade/chem_grenade/incendiary,
 					/obj/item/grenade/chem_grenade/incendiary,
-					/obj/item/grenade/chem_grenade/incendiary)
+					/obj/item/grenade/chem_grenade/incendiary,
+					/obj/item/clothing/suit/fire/atmos,
+					/obj/item/clothing/head/hardhat/atmos)
```

Comment:
Why tho. There's already a firesuit crate

Reply:
Thats for sec, so they can use it without dieing form fire...